src/database/models/user_item.rs

```diff
@@ -298,13 +298,27 @@ impl User {
     pub async fn get_projects<'a, E>(
         user_id: UserId,
         exec: E,
-    ) -> Result<Vec<ProjectId>, sqlx::Error>
+        redis: &deadpool_redis::Pool,
+    ) -> Result<Vec<ProjectId>, DatabaseError>
     where
         E: sqlx::Executor<'a, Database = sqlx::Postgres> + Copy,
     {
         use futures::stream::TryStreamExt;
 
-        let projects = sqlx::query!(
+        let mut redis = redis.get().await?;
+
+        let cached_projects = cmd("GET")
+            .arg(format!("{}:{}", USERS_PROJECTS_NAMESPACE, user_id.0))
+            .query_async::<_, Option<String>>(&mut redis)
+            .await?
+            .and_then(|x| serde_json::from_str::<Vec<ProjectId>>(&x).ok());
+
+        if let Some(projects) = cached_projects {
+            log::info!("Returning cached!");
```

Comment:
I wouldn't mind logging these kinds of things but feels weird to only have the log for this one

Reply:
Whoops, good catch! That was there just for my own reassurance during testing. Didn't intend for it to stay. Will remove